src/servant/tt_000/bat.c

```diff
@@ -479,13 +479,13 @@ void SwitchModeInitialize(Entity* self) {
                 s_BatPathingPoints[self->ext.bat.batIndex][i].y =
                     PLAYER.posY.i.hi + self->ext.bat.cameraY;
             }
-            self->ext.bat.unkA8 = 0;
+            self->ext.bat.hasShotFireball = 0;
```

Comment:
Can this be `false` / `true` also to communicate intent more clearly?

Reply:
Yes, this should use true/false. I’ll fix that.